GOVERNANCE.md

```diff
@@ -0,0 +1,192 @@
+
+Each repository's current list of core-contributors is published and updated in each repo’s OWNERS.md file.
+
+### Maintainer
+
+Each repository in the Atlantis organization are allowed their own unique set of maintainers. Maintainers have the most experience with the given repo and are expected to have the knowledge and insight to lead its growth and improvement.
+
+New maintainers and subproject maintainers must be nominated by an existing maintainer and must be elected by a supermajority of existing maintainers. Likewise, maintainers can be removed by a supermajority of the existing maintainers or can resign by notifying one of the maintainers.
+
+If a Maintainer feels she/he can not fulfill the "Expectations from Maintainers", they are free to step down.
+
+In general, adding and removing maintainers for a given repo is the responsibility of the existing maintainer team for that repo and therefore does not require approval from the steering committee. However, in rare cases, the steering committee can veto the addition of a new maintainer by following the conflict resolution process.
+
+Responsibilities include:
+
+- Strong commitment to the project
+- Participate in design and technical discussions
+- Participate in the conflict resolution and voting process at the repository scope when necessary
+- Seek review and obtain approval from the steering committee when making a change to central architecture that will have broad impact across multiple repositories
+- Contribute non-trivial pull requests
+- Perform code reviews on other's pull requests
+- Ensure that proposed changes to your repository adhere to the established standards, best practices, and guidelines, and that the overall quality and integrity of the code base is upheld.
+- Add and remove maintainers to the repository as described below
+- Approve and merge pull requests into the code base
+- Regularly triage GitHub issues. The areas of specialization possibly listed in OWNERS.md can be used to help with routing an issue/question to the right person.
+- Make sure that ongoing PRs are moving forward at the right pace or closing them
+- Monitor Atlantis Slack (delayed response is perfectly acceptable), particularly for the area of your repository
+- Regularly attend the recurring community meetings
+- Periodically attend the recurring steering committee meetings to provide input
+- In general, continue to be willing to spend at least 25% of their time working on Atlantis (~1.25 business days per week)
+
+Each repository's current list of maintainers is published and updated in each repo’s OWNERS.md file.
+
+#### Removing a core-contributor or maintainer
+
+If a core-contributor or maintainer is no longer interested or cannot perform the the duties listed above, they should volunteer to be moved to emeritus status. In extreme cases this can also occur by a vote of the maintainers per the voting process below.
```

Comment:
```suggestion
If a core-contributor or maintainer is no longer interested or cannot perform the the duties listed above, they should volunteer to be moved to emeritus status. In extreme cases this can also occur by a vote of the maintainers per the voting process below.
```

On paper, contributor/maintainer should opt out, but it is a bit tricky process.

Reply:
Is there any change that needs to be made here? `Volunteer to emerius` is the same as opting out or stepping down. Do we need to improve the language?